packages/ui/Stepper.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+
+type DefaultStep = {
+  title: string;
+};
+
+function Stepper<T extends DefaultStep>(props: { href: string; step: number; steps: T[] }) {
```

Comment:
Moved this component from `console` to `ui`, after this PR I will change`console` to use it from `ui` too.